docs/source/upcoming_release_notes/933-lamp_lv_17.rst

```diff
@@ -0,0 +1,30 @@
+933 lamp_lv_17
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- Adds a new class to interface with the LAMP motion configuration for LV17.
+
+New Devices
+-----------
+- N/A
+
+Bugfixes
+--------
+- N/A
+
+Maintenance
+-----------
+- Mobosum
```

Comment:
```suggestion
- N/A
```

Your name just goes below in Contributors :)

Reply:
I guess I'll fix this when we make the tag...